tests/test_digest_inputs.py

```
@@ -0,0 +1,550 @@
+    as_of_1 = datetime(2026, 5, 13, 12, 0, tzinfo=ZoneInfo("UTC"))
+    as_of_2 = datetime(2026, 5, 14, 12, 0, tzinfo=ZoneInfo("UTC"))
+    conn.close()
+    r1 = collect_digest_inputs(vault_dir, "research-tech", as_of=as_of_1, config=utc_config)
+    r2 = collect_digest_inputs(vault_dir, "research-tech", as_of=as_of_2, config=utc_config)
+    assert r1.input_hash() != r2.input_hash()
+
+
+def test_input_hash_changes_when_data_changes(vault, utc_config):
+    """Adding an evergreen revision should change the hash even
+    within the same window."""
+    vault_dir, conn = vault
```

Ruff (0.15.12)

[warning] Unpacked variable `conn` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)